contributing/documentation/content_guidelines.rst

```diff
@@ -0,0 +1,164 @@
+| The *content* in this section describes the upcoming content from a **business point  |
+| of view**, and shouldn't put the emphasis on Odoo, as this is documentation and not   |
+| marketing.                                                                            |
+|                                                                                       |
+| Start first with a **Lead Paragraph**, which helps the reader make sure that they've  |
+| found the right page, then explain the **business aspects of this topic** in the      |
+| following paragraphs.                                                                 |
++-----+---------------------------------------------------------------------------------+
+|     | | **H2: Section Title (configuration)**                                         |
+|     | | This first H2 section is about the configuration of the feature, or the       |
+|     |   prerequisites to achieve a specific goal. To add a path, make sure you        |
+|     |   use the ``:menuselection:`` specialized directive (see link below).           |
+|     |                                                                                 |
+|     | | Example:                                                                      |
+|     | | To do so, go to ``:menutselection:`App name --> Menu --> Sub-menu```, and     |
+|     |   enable the XYZ feature.                                                       |
++-----+---------------------------------------------------------------------------------+
+|     | | **H2: Section Title (main sections)**                                         |
+|     | | Create as many main sections as you have actions or features to distinguish.  |
+|     |   The title can start with a verb, but try to avoid using "Create ...".         |
++-----+-----+---------------------------------------------------------------------------+
+|     |     | | **H3: Subsection**                                                      |
+|     |     | | Subsections are perfect for assessing very specific points. The title   |
+|     |     |   can be in the form of a question, if appropriate.                       |
++-----+-----+---------------------------------------------------------------------------+
+|     | **H2: Next Section**                                                            |
++-----+---------------------------------------------------------------------------------+
+
+.. seealso::
+   - :ref:`RST cheat sheet: headings <contributing/headings>`
+   - :ref:`RST cheat sheet: specialized directives <contributing/specialized-directives>`
+
+Images
+======
+
+Adding a few images to illustrate your text helps the readers to understand and memorize your
+content. However, avoid adding too many images: it isn't necessary to illustrate all steps and
+features, and it may overload your page.
+
+.. important::
+   Don't forget to compress your PNG files with :ref:`pngquant <contributing/pngquant>`.
```

Comment:
```suggestion
   Don't forget to :ref:`compress your PNG files with pngquant <contributing/pngquant>`.
```
Nitpicking, but I feel that when the link is on 'pngquant' it should lead to a definition of the term.